doc/terminology.rst

```diff
@@ -104,3 +104,11 @@ complete examples, please consult the relevant documentation.*
         one, it has 0 dimensions. That means that, e.g., :py:class:`int`,
         :py:class:`float`, and :py:class:`str` objects are "scalar" while
         :py:class:`list` or :py:class:`tuple` are not.
+
+    duck array
+        `Duck arrays <duck array>`_ are array implementations that behave
```

Comment:
I get a 404 from this link when I try to view the new page on the readthedocs PR build. Are we sure this is right?

Reply:
thanks for catching this, it seems we can't use the `text <label>` syntax in combination with labels. Should be fixed now.